changes/792-aviramha.md

```diff
@@ -0,0 +1,2 @@
+* Change schema and schema_model to handle dataclasses by using their __pydantic_model__ feature.
```

Comment:
```suggestion
* Change schema and schema_model to handle dataclasses by using their `__pydantic_model__` feature.
```
Also, this should be one line and not include stars at the beginning.

Reply:
You've still missed my suggested change here.

Reply:
Ah, sorry. Fixed it now :)